website/public/js/services/taskServices.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 (function(){
-  var TASK_KEYS_TO_REMOVE = ['_id', 'completed', 'date', 'dateCompleted', 'dateCreated', 'history', 'id', 'streak'];
+  var TASK_KEYS_TO_REMOVE = ['_id', 'completed', 'date', 'dateCompleted', 'dateCreated', 'history', 'id', 'streak', 'createdAt'];
```

Comment:
Okay, I changed createAt in the tests. I wasn't sure if I should remove it from this array for any v2 support.